spec/features/idv/doc_auth/verify_info_step_spec.rb

```diff
@@ -151,30 +150,82 @@
       expect(page).to have_current_path(idv_doc_auth_verify_step)
     end
 
-    it 'throttles resolution and continues when it expires' do
-      expect(fake_attempts_tracker).to receive(:idv_verification_rate_limited)
-      sign_in_and_2fa_user
-      complete_doc_auth_steps_before_ssn_step
-      fill_out_ssn_form_with_ssn_that_fails_resolution
-      click_idv_continue
-      (max_attempts - 1).times do
+    context 'resolution throttling' do
+      let(:max_resolution_attempts) { Throttle.max_attempts(:idv_resolution) }
```

Comment:
should/could we stub this to be lower? so the tests don't run too long?

Reply:
Good point. Changed here: eef3500f68adb569d4d1869977bcec11959e4d65